mappings/net/minecraft/structure/pool/SinglePoolElement.mapping

```diff
@@ -17,5 +20,8 @@ CLASS net/minecraft/class_3781 net/minecraft/structure/pool/SinglePoolElement
 		ARG 3 keepJigsaws
 	METHOD method_27233 getStructure (Lnet/minecraft/class_3485;)Lnet/minecraft/class_3499;
 		ARG 1 structureManager
+	METHOD method_28877 locationCodec (Lcom/mojang/datafixers/util/Either;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
 	METHOD method_28878 (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 		ARG 0 instance
+	METHOD method_28880 processorsCodec ()Lcom/mojang/serialization/codecs/RecordCodecBuilder;
```

Comment:
Record based codec consists of a group of getters and a constructor. Since this method only defines a getter,
```suggestion
	METHOD method_28880 processorsGetter ()Lcom/mojang/serialization/codecs/RecordCodecBuilder;
```